src/utils/CommandUtils.hx

```diff
@@ -318,7 +318,7 @@ class CommandUtils
 			while (true)
 			{
 				var line:String = proc.stdout.readLine();
-				if (line == "-D " + Name)
+				if (StringTools.startsWith(line,"-D " + Name + "="))
```

Comment:
Adding the `+ "="` at the end leads to this breaking with Haxe versions earlier than 3.2.0, btw.